package/MDAnalysis/transformations/positionaveraging.py

```diff
@@ -20,34 +39,73 @@
 
 
 
-class PositionAverager:  
+class PositionAverager(object):  
     """
-    
-    Averages the coordinates of a given timestep with the N previous frames. 
-    For frames < N, the average of the frames iterated up to that point will be
-    returned.  
+   
+    Averages the coordinates of a given timestep so that the coordinates
+    of the given atomgroup correspond to the average of the positions 
+    of the N previous frames. 
+    For frames < N, the average of the frames iterated up to that point will
+    be returned.  
     
     Example
     -------
+
+    e.g. Average the coordinates of a given AtomGroup over the course of
+    the previous N frames. For N=3, the output will correspond to the
+    average of the coordinates over the last 3 frames.
+    When check_reset=True, the averager will be reset once the iteration is
+    complete, or if the frames iterated are not sequential.
+
+    .. code-block:: python
+        
+        N=3
+        transformation = PositionAverager(N, check_reset=True)
+        u.trajectory.add_transformations(transformation)   
+        for ts in u.trajectory:
+            print(ts.positions)
     
-    u.trajectory.add_transformations(PositionAverager(N, check_reset=True))    
+    e.g. When check_reset=False, the average of coordinates from non
+    sequential timesteps can also be computed. However, the averager must be
+    manually reset before restarting an iteration.
+           
+    .. code-block:: python
+        
+        N=3
+        transformation = PositionAverager(N, check_reset=False)
+        u.trajectory.add_transformations(transformation)
+        frames = [0, 7, 1, 6]        
+        transformation.resetarrays()
+        for ts in u.trajectory[frames]:
+            print(ts.positions)
```

Comment:
Briefly explain what you expect to see when this code runs. I read the docs and it is not evident to me what the example code will do. Make it easy for the reader, don't expect them to deduce what should happen. Tell them. This is important when someone tries out the code. They want to have reassurance that they get the right thing when they try it for themselves.